cluster/examples/kubernetes/ceph/pool-test.yaml

```
@@ -9,5 +9,6 @@ metadata:
   name: replicapool
   namespace: rook-ceph # namespace:cluster
 spec:
+  failureDomain: host
```

In the pool-test.yaml how about OSD in case the replication is increased to test on a single node with multiple OSDs.
```suggestion
  failureDomain: osd
```